ParquetSharp.DataFrame/ParquetSharp.DataFrame.csproj

```diff
@@ -16,16 +16,11 @@
     <Copyright>Copyright G-Research 2021</Copyright>
     <PackageProjectUrl>https://github.com/G-Research/ParquetSharp.DataFrame</PackageProjectUrl>
     <PackageTags>dataframe apache parquet gresearch g-research .net c#</PackageTags>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/G-Research/ParquetSharp.DataFrame</RepositoryUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
   </PropertyGroup>
 
-  <ItemGroup Label="Additional nuget files">
```

Comment:
Can you please keep the license file included in the NuGet package

Reply:
done